react/src/components/organisms/AccordionWrapper/index.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './style.css';
+
+const AccordionWrapper = (props) => {
+  const children = props.children;
+  return(
+    <div className="ma__accordion-wrapper">
+      { React.Children.map(children, (child) => {
+        if (React.isValidElement(child) && child.type.name === 'AccordionItem') {
+          const clone = React.cloneElement(child, {
+            secondary: props.secondary,
+            emphasize: props.emphasize,
+            border: props.border,
+            headerLevel: props.headerLevel
+          });
+          return clone;
+        }
+        return(
+          /* eslint-disable no-console */
+          console.log(`Warning! You cannot pass a ${child.type.name} child to AccordionWrapper`)
+          /* eslint-disable no-console */
```

Comment:
I think this line was supposed to be `eslint-enable`

Reply:
why should this be `enable`?

Reply:
That or line 20 was maybe supposed to be `eslint-disable-next-line` instead of just disable